dev-tools/build_release.py

```diff
@@ -466,7 +466,7 @@ def smoke_test_release(release, files, expected_hash, plugins):
           if version['build_hash'].strip() !=  expected_hash:
             raise RuntimeError('HEAD hash does not match expected [%s] but got [%s]' % (expected_hash, version['build_hash']))
           print('  Running REST Spec tests against package [%s]' % release_file)
-          run_mvn('test -Dtests.cluster=%s -Dtests.class=*.*RestTests' % ("127.0.0.1:9300"))
```

Comment:
Oops, package is before verify, so I guess we would need to run this manually in the release script, but still it seems this could take the place of starting up and running rest tests externally (but again, as a follow up).